pipeline/frontend/yaml/config.go

```diff
@@ -10,19 +10,21 @@ import (
 type (
 	// Config defines a pipeline configuration.
 	Config struct {
-		Cache     types.Stringorslice
-		Platform  string
-		Branches  constraint.List
-		Workspace Workspace
-		Clone     Containers
-		Pipeline  Containers
-		Services  Containers
-		Networks  Networks
-		Volumes   Volumes
-		Labels    types.SliceorMap
-		DependsOn []string `yaml:"depends_on,omitempty"`
-		RunsOn    []string `yaml:"runs_on,omitempty"`
-		SkipClone bool     `yaml:"skip_clone"`
+		// TODO: change to When (also in Pipeline)
```

Comment:
What would you like to change to When? Constraints => When?

Reply:
Yea. Since you write that in the yaml as when, it should be that in the config. Otherwise its confusing. Also, the yaml parsing dose not follow standard yaml. I made an issue about that here: https://github.com/woodpecker-ci/woodpecker/issues/771